website/docs/reference/warehouse-profiles/bigquery-profile.md

```diff
@@ -103,11 +173,11 @@ my-bigquery-db:
 
 </File>
 
-## Oauth Authorization
 
-To connect to BigQuery using the `oauth` method, follow these steps:
 
+## OAuth Authorization (local)
```

Comment:
Is this the same as `### OAuth Local Authentication`

Reply:
renamed to distinguish